.github/workflows/run-archiver.yml

```diff
@@ -160,3 +155,19 @@ jobs:
           payload: ${{ steps.all_summaries.outputs.SLACK_PAYLOAD }}
         env:
           SLACK_BOT_TOKEN: ${{ secrets.PUDL_DEPLOY_SLACK_TOKEN }}
+
+  make-github-issue:
+    if: always() && inputs.create_github_issue != false
```

Comment:
See discussion here for use of `always()` https://github.com/actions/runner/issues/491

Reply:
I think that's a great idea, can incorporate it here.